### PR TITLE
Fix cht js errors in docker image, update readme for same

### PR DESCRIPTION
**Pull request by mrjones-plip**
# Description

Fixes JS error by updating to ubuntu 16 -> 18, node 6 -> 12, per #448

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Tested: Unit and/or integration tests where appropriate
- Backwards compatible: Works with existing data and configuration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,18 @@ CHT Conf is a command-line interface tool to manage and configure your apps buil
...
 	docker build -t cht-conf:v0 .
-	docker run cht-conf:v0
-	docker exec -it <container_name> /bin/bash
+	docker run -d --name cht-conf cht-conf:v0
+
+You now have a container running in the background called `cht-conf`. You can get a shell on this container with:
+
+	docker exec -it cht-conf /bin/bash
+
+And then you can run the utility with `cht`.  You will have to get your config files into the container in order to be able use the `cht` call to interact with a remote CHT instance.
```
Comment (dianabarsan) [inline README.md:23]: wow ... I understand why nobody is using this :D

File: build.log [hunk 2]
```diff
@@ -0,0 +1,1473 @@
+Sending build context to Docker daemon  8.685MB
```
Comment (dianabarsan) [inline build.log:1]: Was this committed by accident? 
I don't think we should be storing logs on github :)

**Review by dianabarsan — COMMENTED** [review]
Just one comment left about possibly accidentally committed build logs.

File: README.md (hunk 1 above)
Comment (mrjones-plip) [reply README.md:23]: :shrug:

File: build.log (hunk 2 above)
Comment (mrjones-plip) [reply build.log]: Yikes! Thanks, it was for [the ticket](https://github.com/medic/cht-conf/issues/448#issue-1048017825), so already public - but yes, was not meant to be committed, thank you! Fixed in 9373056

**Review by dianabarsan — APPROVED** [review]
Thanks for the fix!

**Comment by mrjones-plip** [comment]
@dianabarsan - I realized after the product call this morning when I went back in to update the `readme`, that we were on an LTS version of node that wasn't supported (`8`) even after upgrading it in this PR. I've bumped it up to `12` in c827736.

Another PR review when you get a sec :pray:

File: README.md [hunk 3]
```diff
@@ -11,29 +11,44 @@ CHT Conf is a command-line interface tool to manage and configure your apps buil
...
+### Windows
 
 As Administrator:
 
 	npm install -g cht-conf
 	python -m pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic --upgrade
 
+### Docker
+
+**NB** - `cht-conf` in a Docker container should only be used if your familure with running containers.  You will have to get your config files into the container in order to be able to use the `cht` call to interact with a remote CHT instance.
```
Comment (dianabarsan) [inline README.md:35]: typo `your familure`
```suggestion
**NB** - `cht-conf` in a Docker container should only be used if you're familiar with running containers. You will have to copy or mount your config files into the container in order to be able to use the `cht` call to interact with a remote CHT instance.
```

**Review by dianabarsan — APPROVED** [review]
Nice! I left one typo fix request inline.

**Review by dianabarsan — APPROVED** [review]
Nice!

**Pull request merged**